contentgrid-spring-boot-autoconfigure/src/main/java/com/contentgrid/spring/boot/autoconfigure/messaging/ContentGridMessagingSystemProperties.java

```diff
@@ -0,0 +1,10 @@
+package com.contentgrid.spring.boot.autoconfigure.messaging;
+
+import java.util.UUID;
+import lombok.Data;
+
+@Data
+public class ContentGridMessagingSystemProperties {
+    private String deploymentId = new UUID(0, 0).toString();
+    private String applicationId = new UUID(0, 0).toString();
```

Comment:
Shouldn't we leave them set to null if they are not set?
And then don't send the application/deployment id headers?